### PR TITLE
Use SSH for github gems

### PR DESCRIPTION
**Pull request by gjvis**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,9 @@ source 'https://rubygems.org'
 
 ruby '2.2.2'
 
+# force Bundler to use HTTPS for github repos
```
Comment (stevenwilkin) [inline Gemfile]: This forces SSH rather than the git protocol, no?

**Comment by andrewgarner** [comment]
:+1:

File: Gemfile (hunk 1 above)
Comment (andrewgarner) [inline Gemfile]: Good spot

**Comment by andrewgarner** [comment]
When the comment is correct, then :+1:

File: Gemfile (hunk 1 above)
Comment (gjvis) [inline Gemfile]: Yep, copied and pasted from another repo (with slightly different constraints) in haste :(

**Pull request merged**